### PR TITLE
Progressive coding conventions update (new branch)

### PR DESCRIPTION
**Pull request by M374LX**

### CODE REVIEW

**Comment by tresf** [comment]
Well, usually we update the original PR instead of create a new one, but that's ok, we can close out the other if you like. 

Seems this is still in conflict with some recent changes tho.  Please resolve these and let us know when and we should be able to merge.

File: include/Note.h [hunk 1]
```diff
@@ -209,8 +209,7 @@ class EXPORT Note : public SerializingObject
 
 
 protected:
-	virtual void saveSettings( QDomDocument & _doc,
-							QDomElement & _parent );
+	virtual void saveSettings( QDomDocument & doc, QDomElement & parent );
 	virtual void loadSettings( const QDomElement & _this );
```
Comment (teeberg) [inline include/Note.h]: `_this` seems badly named. I realise you didn't introduce it, but why not fix it too while you're on it. :-)

File: include/Plugin.h [hunk 2]
```diff
@@ -142,13 +142,15 @@ class EXPORT Plugin : public Model, public JournallingObject
 	typedef QList<Descriptor> DescriptorList;
 
 	// contructor of a plugin
-	Plugin( const Descriptor* descriptor, Model* parent );
+	Plugin( const Descriptor * descriptor, Model * parent );
```
Comment (teeberg) [inline include/Plugin.h]: As per recent discussions, I think the conclusion is to use either `type* var` or `type *var`.

File: include/Song.h [hunk 3]
```diff
@@ -49,10 +49,9 @@ const tick_t MaxSongLength = 9999 * DefaultTicksPerTact;
 class EXPORT Song : public TrackContainer
 {
 	Q_OBJECT
-	mapPropertyFromModel(int,getTempo,setTempo,m_tempoModel);
-	mapPropertyFromModel(int,masterPitch,setMasterPitch,m_masterPitchModel);
-	mapPropertyFromModel(int,masterVolume,setMasterVolume,
-							m_masterVolumeModel);
+	mapPropertyFromModel( int,getTempo,setTempo,m_tempoModel );
+	mapPropertyFromModel( int,masterPitch,setMasterPitch,m_masterPitchModel );
+	mapPropertyFromModel( int,masterVolume,setMasterVolume, m_masterVolumeModel );
```
Comment (teeberg) [inline include/Song.h]: Maybe add spaces between the arguments?

File: include/Song.h [hunk 4]
```diff
@@ -100,9 +99,9 @@ class EXPORT Song : public TrackContainer
 	{
 		return m_elapsedMilliSeconds;
 	}
-	inline void setMilliSeconds( float _ellapsedMilliSeconds )
+	inline void setMilliSeconds( float ellapsedMilliSeconds )
```
Comment (teeberg) [inline include/Song.h]: ellapsed -> elapsed

File: include/Timeline.h [hunk 5]
```diff
@@ -60,11 +60,11 @@ class Timeline : public QWidget, public JournallingObject
 	} ;
 
 
-	Timeline( int _xoff, int _yoff, float _ppt, Song::playPos & _pos,
+	Timeline( int _xoff, int _yoff, float _ppt, Song::PlayPos & _pos,
 				const MidiTime & _begin, QWidget * _parent );
```
Comment (teeberg) [inline include/Timeline.h]: Remove the underscores of the arguments here too?

File: src/core/Song.cpp [hunk 6]
```diff
@@ -539,12 +553,14 @@ void Song::updateLength()
...
+void Song::setPlayPos( tick_t ticks, PlayModes playMode )
 {
-	m_elapsedTicks += m_playPos[_play_mode].getTicks() - _ticks;
-	m_elapsedMilliSeconds += (((( _ticks - m_playPos[_play_mode].getTicks()))*60*1000/48)/getTempo());
-	m_playPos[_play_mode].setTicks( _ticks );
-	m_playPos[_play_mode].setCurrentFrame( 0.0f );
+	m_elapsedTicks += m_playPos[playMode].getTicks() - ticks;
+	m_elapsedMilliSeconds += 
+		( ( ( ( ticks - m_playPos[playMode].getTicks() ) ) * 60 * 1000 / 48) / 
+			getTempo() );
```
Comment (teeberg) [inline src/core/Song.cpp]: This kind of expression could use another local variable.

File: src/core/Timeline.cpp [hunk 7]
```diff
@@ -51,7 +51,7 @@ QPixmap * Timeline::s_loopPointBeginPixmap = NULL;
 QPixmap * Timeline::s_loopPointEndPixmap = NULL;
 
 Timeline::Timeline( const int _xoff, const int _yoff, const float _ppt,
-			Song::playPos & _pos, const MidiTime & _begin,
+			Song::PlayPos & _pos, const MidiTime & _begin,
```
Comment (teeberg) [inline src/core/Timeline.cpp]: More arguments starting with `_`.

File: src/core/Track.cpp [hunk 8]
```diff
@@ -711,17 +711,17 @@ void TrackContentObjectView::mousePressEvent( QMouseEvent * _me )
  *  * or if in resize mode, resize ourselves,
  *  * otherwise ???
  *
- * \param _me The QMouseEvent to handle.
+ * \param me The QMouseEvent to handle.
  * \todo what does the final else case do here?
  */
-void TrackContentObjectView::mouseMoveEvent( QMouseEvent * _me )
+void TrackContentObjectView::mouseMoveEvent( QMouseEvent * me )
```
Comment (teeberg) [inline src/core/Track.cpp]: Wouldn't `event` be a better name for most events?

**Comment by M374LX** [comment]
The merge conflicts seem to be solved.

@teeberg 
I will review your comments later, as well as the comments on #1559.

**Comment by tresf** [comment]
@M374LX as you can expect with the amount of files this touches, this requires yet another conflict fix.  I'd really like @Lukas-W to merge this one as he's been helping with improving our coding standards.

**Comment by tresf** [comment]
@M374LX do you have time to update this to fix the merge conflicts?

**Comment by M374LX** [comment]
I am going to check this out as soon as I get less busy.

2015-02-26 11:46 GMT-03:00 Tres Finocchiaro <notifications@github.com>:

**Comment by tresf** [comment]
I'm sorry I hadn't noticed this before, we have changed our formatting slightly for parenthesis:

https://github.com/LMMS/lmms/wiki/Coding-conventions

**Comment by M374LX** [comment]
@tresf 
I suggest to merge this PR now, so we can check the parentheses later.

**Comment by tresf** [comment]
Thanks.  Yeah, `ProjectVersion` was the regression, which you've addressed. :+1:   Merging.

**Comment by Spekular** [comment]
ProjectVersion? Oops, sorry about that!

**Pull request merged**